routes/GET-post.js

```diff
@@ -30,7 +30,7 @@ module.exports = (app, db) => {
 
 			db.query(`SELECT email FROM cloudclub.logins WHERE id='${result[0].userid}'`, (err2, result2) => {
 				if (err2) { console.log(err2); return res.status(500).json({ error: "Internal Server Error 500" }); }
-				return res.render("post", { "selected": "forum", "title": "CloudClub | Forum", "post": result[0], "email": result2[0].email, "user_id":req.session.userid });
+				return res.render("post", { "selected": "forum", "title": "CloudClub | Forum", "post": result[0], "email": result2[0].email, "user_id":req.session.userid, "profile_path": result2[0].profile_path });
```

Comment:
Hi @st-C0nst ,

Please ensure that the correct table is being queried for the profile_path variable. The logins table does not contain the URL of the profile picture. This information is stored inside the profiles table. 

Here `results2[0].profile_path` will **always** be undefined because the logins table is being queried and any row inside it will not contain the profile_path column.